src/api/ledger/parse/suspended-payment-finish.js

```diff
@@ -0,0 +1,25 @@
+/* @flow */
+'use strict';
+const assert = require('assert');
+const sjclcodec = require('sjcl-codec');
+const utils = require('./utils');
+
+function convertHexToString(hexString) {
+  const bits = sjclcodec.hex.toBits(hexString);
+  return sjclcodec.utf8String.fromBits(bits);
+}
+
+function parsePayment(tx: Object): Object {
+  assert(tx.TransactionType === 'SuspendedPaymentFinish');
+
+  return utils.removeUndefined({
+    memos: utils.parsePaymentMemos(tx),
+    owner: tx.Owner,
+    offerSequence: tx.OfferSequence,
```

Comment:
Given that this is already an uncommon field, adding another uncommon field is not a big deal. The problem is that "OfferSequence" should have been named something else from the start -- maybe "OtherSequence" or something. So if we add a new field, it will wind up being "PaymentSequence" and we'll multiply the bad choice. But I think that's about the best we can do.